Formula/stellar-core.rb

```diff
@@ -0,0 +1,37 @@
+class StellarCore < Formula
+  desc "The backbone of the Stellar (XLM) network"
+  homepage "https://www.stellar.org/"
+  url "https://github.com/stellar/stellar-core.git",
+    :using => :git,
```

Comment:
The `:using => :git` here and below are both redundant.

Reply:
Done, thanks.